sdk/core/azure-core-http-jdk-httpclient/src/test/java/com/azure/core/http/jdk/httpclient/JdkAsyncHttpClientTests.java

```diff
@@ -221,8 +221,8 @@ public void testConcurrentRequests() throws NoSuchAlgorithmException {
                 // Thread.currentThread().getName()))
                 .map(nbb -> (long) nbb.bb.limit())
                 .reduce(Long::sum)
-                .subscribeOn(reactor.core.scheduler.Schedulers.newElastic("io", 30))
-                .publishOn(reactor.core.scheduler.Schedulers.newElastic("io", 30));
+                .subscribeOn(Schedulers.newBoundedElastic(30, 1024, "io"))
```

Comment:
Same here - don't need both subscribeOn and publishOn. Just publishOn is sufficient.

Reply:
Will clean up